include/s1ap/3gpp_s1ap_36413.h

```diff
@@ -0,0 +1,36 @@
+/*
+* Copyright (c) 2003-2020, Great Software Laboratory Pvt. Ltd.
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+#ifndef __S1AP_MACROS_H_
+#define __S1AP_MACROS_H_
+
+#include <stdbool.h>
+
+/*Refrence document 3GPP TS 36.413 (http://www.arib.or.jp/english/html/overview/doc/STD-T104v3_00/5_Appendix/Rel12/36/36413-c50.pdf) */
+enum e_rab_struct_val
+{
+  E_RAB_ID = 1, //the E-RAB ID uniquely identifies an E-RAB for one UE
```

Comment:
Again seeing this..I am not convinced that all these values can be part of the same enum.
Can you please point me to the section of 36.413 which has these values defined.

Reply:
9.1.3.1 E-RAB SETUP REQUEST -- for RAB ID and QoS structure 
9.2.1.15 E-RAB Level QoS Parameters - for param QCI which again points to "Allocation and Retention
Priority" structure  (9.2.1.60 Allocation and Retention Priority) .
It is like structure inside structure

Reply:
How can enum having same value multiple variables ??? Something wrong.. 

e.g. QCI can have many values. 9 is one of the value.  Instead of having QCI enum, you are using this enum with QCI=9. this is wrong.